README.md

```diff
@@ -22,22 +27,30 @@ with pm.Model():
 
 ### What belongs in `pymc-experimental`?
 
-- newly-implemented statistical methodologies
+- newly-implemented statistical methods, for example diagnostics or step methods
 - distributions that are tricky to sample from or test
 - infrequently-used fitting methods or distributions
 - any code that requires additional optimization before it can be used in practice
 
 
 ### What does not belong in `pymc-experimental`?
+- Case studies
+- Implementations that cannot be applied generically
```

Comment:
we may want to clarify this. I guess the idea is that we should support features that agree with "probabilistc programming paradigm", but all methods are restricted in one way or another

Reply:
```suggestion
- Implementations that cannot be applied generically, for example because they are tied to variables from a toy example
```